package.json

```diff
@@ -255,6 +256,7 @@
     "react-test-renderer": "^16.12.0",
     "read-installed": "^4.0.3",
     "redux-mock-store": "^1.5.4",
+    "regenerator-runtime": "^0.13.3",
```

Comment:
😭

Reply:
Yeah it sucks. This is fixed in v6 of Storybook already, but they didn't backport it. We can remove it with the v6 upgrade, whenever that's ready.